app/jobs/regular/user_email.rb

```diff
@@ -34,11 +34,13 @@ def execute(args)
       # of extra work when emails are disabled.
       return if quit_email_early?
 
+      args[:type] = ActiveSupport::StringInquirer.new(args[:type].to_s)
```

Comment:
TIL about `StringInquirer`

Reply:
Ok in the end I went for the `#to_s` road, so we keep things simple. I’ve started a bigger refactoring here: https://github.com/discourse/discourse/pull/20613 (it’s clearly not finished yet though :sweat_smile:)